client/public/index.html

```diff
@@ -1,198 +1,11 @@
-              <label for="view_output" title="Show only the output">Output</label>
-            </li>
-            <li class="view_gist_li">
-              <a class="view_gist" target="trypurs_gist">
-                <label title="Open the original gist in a new window">Gist</label>
-              </a>
-            </li>
-          </ul>
-        </li><li class="menu-item view_gist_li mobile-only">
-          <a class="view_gist" target="trypurs_gist">
-            <label title="Open the original gist in a new window">View Gist</label>
-          </a>
-          <!-- TODO: uncomment the below once https://github.com/purescript/trypurescript/issues/118 is resolved -->
-          <!-- </li><li class="menu-item no-mobile"> -->
-          <!-- <label id="gist_save" name="save_gist" title="Save file as a Gist">Share</label> -->
-        </li><li class="menu-item no-mobile">
-          <label id="compile_label" name="compile_label" title="Compile Now">Compile</label>
-        </li><li class="menu-item nowrap no-mobile">
-          <input id="auto_compile" name="auto_compile" title="Toggle auto-compliation of the file on code changes" value="auto_compile" type="checkbox" checked="true">
-          <label id="auto_compile_label" for="auto_compile" title="Compile on code changes">Auto-Compile</label>
-        </li><li class="menu-item nowrap">
-          <input id="showjs" name="showjs" title="Show resulting JavaScript code instead of output" value="showjs" type="checkbox">
-          <label id="showjs_label" for="showjs" title="Show resulting JavaScript code instead of output">Show JS</label>
-        </li><li class="menu-item">
-          <a id="helplink" href="https://github.com/purescript/trypurescript/blob/master/README.md" target="trypurs_readme">
-            <label id="help" title="Learn more about Try PureScript">Help</label>
-          </a>
-        </li>
-      </ul>
-
-      <div class="mobile-only mobile-banner">
-        Your screen size is too small. Code editing has been disabled.
-      </div>
-
-      <div id="editor_view" data-view-mode="sidebyside">
-        <div id="column1" class="no-mobile">
-
-          <div id="code">
-          </div>
-          <textarea name="code" id="code_textarea" style="display: none;"></textarea>
-        </div>
-
-        <div class="separator">
-        </div>
-
-        <div id="column2_wrapper">
-          <div id="column2"></div>
-          <div id="loading"></div>
-        </div>
-      </div>
-    </div>
-  </div>
-
-  <script type="text/javascript">
-    $.ajaxSetup({
-      dataType: 'text'
-    });
-
-    var editor, cleanupActions = [];
-
-    editor = ace.edit('code');
-
-    editor.renderer.setShowGutter(true);
-    editor.setFontSize(13);
-    editor.setShowPrintMargin(false);
-
-    var session = editor.getSession();
-
-    session.setMode('ace/mode/haskell');
-    session.setOptions({
-      tabSize: 2,
-      useSoftTabs: true
-    });
```

Comment:
* showGutter already defaults to true
* original font size is 13, but appears to default to 12. Can't edit this without fixing ace editor library to take a number rather than a string for font size.
* soft tabs enabled by default